lib/workos/user_management.rb

```diff
@@ -678,6 +678,118 @@ def send_password_reset_email(email:, password_reset_url:)
         WorkOS::UserAndToken.new(response.body)
       end
 
+      # Gets an Invitation
+      #
+      # @param [String] id The unique ID of the Invitation.
+      #
+      # @return WorkOS::Invitation
+      sig do
+        params(id: String).returns(WorkOS::Invitation)
+      end
+      def get_invitation(id:)
```

Comment:
All of those methods should be added to the [API reference](https://workos.com/docs/reference/user-management/password-reset/complete) as well.